src/components/Header/Header.css

```diff
@@ -52,5 +52,31 @@
     gap: 10px;
 }
 
+@media screen and (max-width: 425px) {
+    .header {
+       padding: 20px 12px 74px 12px;
```

Comment:
`padding: 20px 12px 74px` should work without additional 12 right? (just stylistic comment, disregard if preferred!)